src/geo/GoogleTiles.ts

```diff
@@ -0,0 +1,64 @@
+import {IGoogleTile, IPoint, IBBox} from "../Geo";
+import {ILatLon} from "../Graph";
+
+export class GoogleTiles {
```

Comment:
This class has some magic numbers that would be easier to understand if they were made constants.
The class can also easily be unit tested to provide documentation of how it works.